code/game/objects/items/twohanded.dm

```diff
@@ -1016,3 +1016,257 @@
 		C.change_view(CONFIG_GET(string/default_view))
 		user.client.pixel_x = 0
 		user.client.pixel_y = 0
+
+/obj/item/twohanded/electrostaff
+	icon = 'icons/obj/estaff.dmi'
+	icon_state = "electrostaff_3"
+	item_state = "electrostaff_3"
+	lefthand_file = 'icons/mob/inhands/weapons/estaff_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/estaff_righthand.dmi'
+	name = "riot suppression electrostaff"
+	desc = "A large quarterstaff, with massive silver electrodes mounted at the end."
+	force = 10
+	damtype = BRUTE
+	w_class = WEIGHT_CLASS_HUGE
+	slot_flags = ITEM_SLOT_BACK | ITEM_SLOT_OCLOTHING
```

Comment:
would you consider adding this to the security armour's whitelist rather than oclothing, seems odd it would be oclothing.

Reply:
if you DO wanna keep it there, add some sort of equipped sprite. it's showing the green SUIT

Reply:
I understand the suit pocket is coded to be unable to hold huge items, but this is not a suit, Kevinz, add a blacklist for the suit pocket size cap instead. Also missing sprites.